apps/sms/test/marionette/lib/views/conversation/accessors.js

```diff
@@ -122,9 +122,15 @@ ConversationAccessor.prototype = {
 
   waitToAppear: function() {
     var conversationPanel = this.client.helper.waitForElement(SELECTORS.main);
-    this.client.waitFor(function() {
-      return conversationPanel.location().x === 0;
+
+    this.client.waitFor(() => {
+      return conversationPanel.location().x === 0 && this.client.executeScript(
```

Comment:
At first I was thinking about using smth like ```getActivePanelName``` from main messages lib, but it seems it doesn't distinguish conversation and composer.

Also I was considering moving this ```executeScript``` part to shared/app_accessors (that doesn't exist yet) that will expose such method with ```panelName``` parameter - but my rule of thumb is not to generalize unless we have more than one usage place :) But tell me what you think!